supernode/daemon/mgr/gc/gc_dfget_task.go

```diff
@@ -25,6 +25,16 @@ import (
 func (gcm *Manager) gcDfgetTasksWithTaskID(ctx context.Context, taskID string, cids []string) []error {
 	var errSlice []error
 	for _, cid := range cids {
+		// these peers may crashes that haven't report sever down.
+		// TODO:now pid and cid correspond one-to-one,so we can delete a pid by cid. but in future,our dragonfly api will make pid and cid one to many,so we can't delete a pid by cid.
+		// TODO:in future we may add heartbeat between dfget and supernode.so we will implement it in other way.
+		dfgetTask, err := gcm.dfgetTaskMgr.Get(ctx, cid, taskID)
+		if err != nil {
+			errSlice = append(errSlice, errors.Wrapf(err, "failed to get dfgetTask by cid(%s)", cid))
+			continue
+		}
+		gcm.progressMgr.UpdatePeerServiceDown(ctx, dfgetTask.PeerID)
+
 		if err := gcm.progressMgr.DeleteCID(ctx, cid); err != nil {
```

Comment:
We should exec the delete cid operation even if failed to get the dfgetTask info.

Reply:
> We should exec the delete cid operation even if failed to get the dfgetTask info.

Done